cardano-sl.cabal

```diff
@@ -12,17 +12,11 @@ build-type:          Simple
 extra-source-files:  README.md
 cabal-version:       >=1.10
 
-Flag with-web
-  default:     True
-  manual:      True
-
-  description: Build with web part.
-
 Flag with-wallet
```

Comment:
@gromakovsky It is needed here https://github.com/input-output-hk/cardano-sl/blob/cb7d38f37c211592f0a38cb1c93096df2ba10108/cardano-sl.cabal#L429 to avoid gold linker issue as described here https://ghc.haskell.org/trac/ghc/ticket/13810